.github/scripts/run_tests.sh

```diff
@@ -5,13 +5,9 @@ if [ "$OS" = "macos" ]; then
     PATH=/Users/runner/miniconda3/envs/test/bin:/Users/runner/miniconda3/condabin:$PATH
 fi
 
-# Rename log file before running the test suite. Its contents will be read
-# before the next run (see conftest.py in the root of the repo).
-mv log.txt pytest_log.txt
-
 # Run tests
 if [ "$OS" = "linux" ]; then
     xvfb-run --auto-servernum python runtests.py --color=yes | tee log.txt
```

Comment:
Why not use `tee -a pytest_log.txt`  here too?

Reply:
Oops!